### PR TITLE
wmco: mark deprecated bundle as optional to remove it from required pipeline contexts

### PR DESCRIPTION
**Pull request by jmguzik**
I implemented this possibility in conf generator.

### CODE REVIEW

File: ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml [hunk 1]
```diff
@@ -31,6 +31,7 @@ operator:
   - as: wmco-bundle
     context_dir: .
     dockerfile_path: bundle.Dockerfile
+    optional: true
```
Comment (sebsoto) [inline ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml:34]: how is this deprecated?

Comment (sebsoto) [reply ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml:34]: This isn't building an index, but a bundle https://docs.ci.openshift.org/docs/how-tos/testing-operator-sdk-operators/#building-operator-bundles

Comment (aravindhp) [reply ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml:34]: @sebsoto If the bundle is not getting built now how are the tests that depend on it running and passing?

Comment (sebsoto) [reply ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml:34]: To be honest I'm a bit confused here. I'm not following where/what caused this to not be built.

**Pull request closed without merging**